docs/contributing/contributing.md

```diff
@@ -1,5 +1,3 @@
 # Contributing overview
 
-The Arm Mbed OS porting guide describes how to a add development board Mbed OS and configure it. It includes porting information about related features, such as cellular, exporters and Mbed TLS. It also contains information about CMSIS, Keil RTX and Thread safety.
```

Comment:
this file isn't actually used as far as I can tell (not in docs.json)

Reply:
I can remove the file, or ?